docs/docs/50-user-guide/60-reference-docs/30-promotion-steps/git-open-pr.md

```diff
@@ -10,15 +10,15 @@ specified source and target branches. This step is often used after a
 [`git-push` step](git-push.md) and is commonly followed by a
 [`git-wait-for-pr` step](git-wait-for-pr.md).
 
-At present, this feature only supports GitHub and Gitea pull requests, and
-GitLab merge requests.
+At present, this feature only supports GitHub, Gitea and Azure DevOps pull
+requests, and GitLab merge requests.
 
 ## Configuration
 
 | Name | Type | Required | Description |
 |------|------|----------|-------------|
 | `repoURL` | `string` | Y | The URL of a remote Git repository. |
-| `provider` | `string` | N | The name of the Git provider to use. Currently only `github`, `gitlab`  and `gitea` are supported. Kargo will try to infer the provider if it is not explicitly specified.  |
+| `provider` | `string` | N | The name of the Git provider to use. Currently only `github`, `gitlab`, `azure` and `gitea` are supported. Kargo will try to infer the provider if it is not explicitly specified.  |
```

Comment:
Slight preference for an Oxford comma, but in the end, I don't care that much. Same in the wait-for-pr step docs.

```suggestion
| `provider` | `string` | N | The name of the Git provider to use. Currently only `github`, `gitlab`, `azure`, and `gitea` are supported. Kargo will try to infer the provider if it is not explicitly specified.  |
```